src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -1065,9 +1065,9 @@ private static String forAgency(final String agencyId) {
 
     public InmateDetail getOffender(final String offenderNo) {
         telemetryClient.trackEvent("getOffender 1.1_beta version used");
-        return offenderRepository.findRootOffenderByNomsId(offenderNo)
+        return offenderBookingRepository.findLatestOffenderBookingByNomsId(offenderNo)
```

Comment:
best way to get the booking is from the booking repository instead, otherwise get the root. Don't think this method is used (will find out in a week or so when the telemetry gets recorded), but have fixed it anyway.